actors/evm/src/lib.rs

```diff
@@ -53,16 +51,32 @@ impl EvmContractActor {
         RT: Runtime<BS>,
     {
         // TODO ideally we would be checking that we are constructed by the EAM actor,
-        //   so instead we check for init and then assert that we have a delegated address.
-        rt.validate_immediate_caller_type(iter::once(&Type::Init))?;
-
-        // BIG LARGE TODO AS IN TODO TOMORROW
-        // - @mriise
-        // if rt.lookup_address(rt.message().caller().id().unwrap()).is_none() {
-        //     return Err(ActorError::assertion_failed(
-        //         "EVM actor created without a delegated address".into(),
-        //     ));
-        // }
+        //   but instead we check for init and then assert that we have a delegated address.
```

Comment:
For example, this is a good TODO (although we should probably link to https://github.com/filecoin-project/ref-fvm/issues/746). When we're browsing through the code in the future, we might decide to fix it.